frontend/src/services/LogHoursAPI.ts

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+import { LogHoursCreationDTO } from '../dto/LogHours/LogHoursDTOs';
+
+export const createLogHours = async (logHoursCreationDTO: LogHoursCreationDTO): Promise<any> => {
```

Comment:
similar to previous comment in AccountAPI.ts, you can specify the promise data type.